softwarereview_policies.Rmd

```diff
@@ -157,6 +157,128 @@ Despite our best efforts to support contributed software, errors are the respons
 
 In the unlikely scenario that a contributor of a package requests removal of their package from the suite, we retain the right to maintain a version of the package in our suite for archival purposes.
 
+## Ethics, Data Privacy and Human Subjects Research
+
+rOpenSci tools are used for a variety of purposes, but our focus is on
```

Comment:
Should tools become packages or should it at least be noted that in the context of the dev guide, tools are packages?